From 8a0e25c18061fb47517c859db82f8de0e45d4203 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 7 Mar 2022 03:10:50 +0000
Subject: libstdc++: fix incomplete type of blink::WebURLLoaderFactory

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of blink::WebURLLoaderFactory in
blink::EmptyChromeClient.

Bug: 957519
Change-Id: I6a7b6aee0f712983cab77c038df1037cd6ad8e25
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3500523
Reviewed-by: Yutaka Hirano <yhirano@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#978123}
---
 third_party/blink/renderer/core/loader/empty_clients.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/renderer/core/loader/empty_clients.h b/third_party/blink/renderer/core/loader/empty_clients.h
index d91e67a5ea2e5..e23de752ca049 100644
--- a/third_party/blink/renderer/core/loader/empty_clients.h
+++ b/third_party/blink/renderer/core/loader/empty_clients.h
@@ -43,6 +43,7 @@
 #include "third_party/blink/public/mojom/input/focus_type.mojom-blink-forward.h"
 #include "third_party/blink/public/platform/web_spell_check_panel_host_client.h"
 #include "third_party/blink/public/platform/web_url_loader.h"
+#include "third_party/blink/public/platform/web_url_loader_factory.h"
 #include "third_party/blink/renderer/core/core_export.h"
 #include "third_party/blink/renderer/core/frame/local_frame_client.h"
 #include "third_party/blink/renderer/core/frame/remote_frame_client.h"
-- 
2.32.0

